build.sbt

```diff
@@ -5,7 +5,7 @@ name := "soy"
 organization := "com.kinja"
 
 // We use Semantic Versioning. See: http://semver.org/
-version := "0.4.4" + {if (System.getProperty("JENKINS_BUILD") == null) "-SNAPSHOT" else ""}
+version := "0.5" + {if (System.getProperty("JENKINS_BUILD") == null) "-SNAPSHOT" else ""}
```

Comment:
From [semver.org](http://semver.org/):
`Major version zero (0.y.z) is for initial development. Anything may change at any time. The public API should not be considered stable.`
I think this means that there's no need for a major version bump when a change is not backward compatible but we're still on 0.x.y